src/osx/carbon/font.cpp

```diff
@@ -295,6 +295,11 @@ wxFontRefData::wxFontRefData(wxOSXSystemFont font, int size)
 
 static const CGAffineTransform kSlantTransform = CGAffineTransformMake( 1, 0, tan(wxDegToRad(11)), 1, 0, 0 );
 
+typedef struct {
+    wxCFRef< CTFontRef > font;
+    wxCFRef< CFDictionaryRef > fontAttributes;
+} OSXFontEntry;
```

Comment:
A nitpick, but perhaps it would be better to use the word "cache" somewhere in the name here? E.g. `OSXCachedFontEntry` (or even just `CachedFontEntry` because there is no risk of a clash if it's declared inside the anon namespace).